rllib/agents/trainer_template.py

```diff
@@ -171,7 +167,7 @@ def with_updates(**overrides):
         """
         return build_trainer(**dict(original_kwargs, **overrides))
 
-    trainer_cls.with_updates = with_updates
+    trainer_cls.with_updates = staticmethod(with_updates)
```

Comment:
Any reason not to decorate the original method?

Reply:
PyCharm marks it as bad style due to the decorator being inside a function, not a class.